embedded-graphics/src/geometry/point.rs

```diff
@@ -78,11 +78,66 @@ impl Point {
         Point { x, y }
     }
 
+    /// Creates a point with X and Y values set to an equal value.
+    ///
+    /// ```rust
+    /// use embedded_graphics::geometry::Point;
+    ///
+    /// let filled = Point::new_filled(11);
+    ///
+    /// assert_eq!(filled, Point { x: 11, y: 11 });
+    /// ```
+    pub const fn new_filled(fill: i32) -> Self {
+        Point { x: fill, y: fill }
+    }
+
     /// Creates a point with X and Y equal to zero.
     pub const fn zero() -> Self {
         Point { x: 0, y: 0 }
     }
 
+    /// Return the `Point` with `y` set to `0`.
+    ///
+    /// # Examples
+    ///
+    /// ## Move a `Point` along the X axis.
+    ///
+    /// ```rust
+    /// use embedded_graphics::geometry::{Point, Size};
+    ///
+    /// let size = Size::new(20, 30);
+    ///
+    /// let point = Point::new(10, 15);
+    ///
+    /// let moved_x = point + size.x_axis();
```

Comment:
This isn't really an example of `Point::x_axis`, but `Size::x_axis`. Because there are no additional tests for these functions the example should use `Point::x_axis`.